polaris.shopify.com/src/components/GlobalSearch/GlobalSearch.tsx

```diff
@@ -52,23 +52,14 @@ function captureSearchEvent(
   if (!searchTerm) return;
 
   const customParams = {
-    searchTerm,
-    resultRank,
-    selectedResult,
+    search_term: searchTerm,
+    result_rank: resultRank,
+    selected_result: selectedResult,
     category: 'engagement',
   };
 
-  const googleParams = {
```

Comment:
we only need one of these events and the custom event is set up with the metrics we want